libs/form/src/next/input-field-masked/stories/vl-input-field-masked.stories-arg.ts

```diff
@@ -1,44 +1,41 @@
-import { CATEGORIES, TYPES } from '@domg-wc/common-storybook';
+import { CATEGORIES, CONTROLS, TYPES, getSelectControlOptions } from '@domg-wc/common-storybook';
 import { ArgTypes } from '@storybook/web-components';
 import { inputFieldMaskedDefaults } from '../vl-input-field-masked.component';
 import { inputFieldArgTypes, inputFieldArgs } from '../../input-field/stories/vl-input-field.stories-arg';
 import { masks } from '../masks';
 
-type InputFieldMaskedArgsType = typeof inputFieldArgs & typeof inputFieldMaskedDefaults;
+type InputFieldMaskedArgs = typeof inputFieldArgs & typeof inputFieldMaskedDefaults;
 
-export const inputFieldMaskedArgs: InputFieldMaskedArgsType = {
+export const inputFieldMaskedArgs: InputFieldMaskedArgs = {
     ...inputFieldArgs,
     ...inputFieldMaskedDefaults,
 };
 
-export const inputFieldMaskedArgTypes: ArgTypes<typeof inputFieldMaskedArgs> = {
+export const inputFieldMaskedArgTypes: ArgTypes<InputFieldMaskedArgs> = {
     ...inputFieldArgTypes,
     pattern: {
         name: 'pattern',
-        description:
-            'Dit attribuut wordt uitgeschakeld bij de input-field-masked component omdat het mask deze werking overneemt.<br/>Gelieve dit attribuut niet te gebruiken.',
-        control: { type: null },
-        table: {
-            type: { summary: TYPES.STRING },
-            category: CATEGORIES.ATTRIBUTES,
-            defaultValue: { summary: inputFieldMaskedArgs.pattern },
-        },
+        if: { arg: '' }, // Dit attribuut mag niet gebruikt worden bij dit component, verberg het in Storybook.
```

Comment:
Op zich als het werkt, is het belangrijkste, maar dit is eerder bedoeld om conditionele controls af te beelden.

De Storybook manier om dit te doen is:

```
CascaderDefault.parameters = {
    controls: {
        exclude: ['content', 'label', 'labelSlot'],
    },
};
```

Dan alles je dit wil toepassen voor alle stories, kan je dit doen:

```
export default {
    ...
    parameters: {
        ...
        controls: { exclude: ['data-vl-hover'] },
    },
} as Meta<typeof dataTableArgs>;
```